atomic_reactor/plugins/post_push_to_pulp.py

```diff
@@ -112,7 +113,9 @@ def push_tarball_to_pulp(self, image_names):
             repos_tags_mapping[repo].setdefault("tags", [])
             repos_tags_mapping[repo]["tags"].append(image.tag)
         self.log.info("repo_tags_mapping = %s", repos_tags_mapping)
-        task_ids = p.push_tar_to_pulp(repos_tags_mapping, self.filename)
+
+        prefix = "scratch-" if self.scratch_build else "redhat-"
```

Comment:
The `redhat-` prefix is hardcoded in dockpulp and is used when no `repo_prefix` is given. You've got a point though - perhaps the plugin can accept the prefix as a parameter instead of the boolean `scratch_build`.